### DISCUSSION TITLE
Migrating from Firebase (cloud functions) to AceBase

### DISCUSSION THREAD

**Discussion by amitnil-dev** [appy-one/acebase#125] [Q&A]
Greetings,

I have recently come across a situation where I need to migrate my webapp(_angular, nodeJs_)  from FIrebase (cloud functions etc).. to MongoDb and I stumbled upon AceBase which by description looks perfect. Whats the best way to go about this.

Thanks
amit....

**Comment by amitnil-dev** [comment]
any suggestion?

**Comment by appy-one** [comment]
Hi Amit, sorry for replying so late, I've been on holiday! 🌴

Awesome you want to migrate to AceBase! I don't know what your particular usecase is, but much of AceBase's SDK is compatible with Firebase's realtime db Javascript SDK. One of the first steps you'd want to take is exporting your data from Firebase, and importing it into either a local AceBase database, or a remote AceBaseServer instance. Since you mentioned cloud functions, you probably want to run a Node.js app in the cloud or your own server: check out how to [setup an AceBaseServer](https://github.com/appy-one/acebase-server). You can use an [AceBaseClient](https://github.com/appy-one/acebase-client) within your Angular app to connect to your server.

If you have more specific questions, don't hesitate to ask them here. I'm happy to help!
